doc/api/cluster.md

```diff
@@ -459,6 +459,11 @@ if (cluster.isMaster) {
 }
 ```
 
+`worker.send()` will return `false` if the channel has closed or when the
```

Comment:
I think it will throw if the channel is closed, could you try?

Reply:
Okay, I am not sure how to try. I will see if there is a unit test for this.

Reply:
I found this test: `test-child-process-fork-closed-channel-segfault.js`. It does look like it throws an error, if the channel is closed. I will update the doc.